src/net/http/Easy.hxx

```diff
@@ -95,6 +96,14 @@ public:
 			throw std::runtime_error(curl_easy_strerror(code));
 	}
 
+#ifdef UNICODE
+	void SetOption(CURLoption option, const TCHAR* value) {
+		CURLcode code = curl_easy_setopt(handle, option, WideToACPConverter(value));
```

Comment:
This is a bad conversion. It uses the ACP charset for communication over the network. Unfortunately, the basic idea of ACP is a historic mistake, it's inherently broken and the actual ACP charset differs depending on your computer's locale.
Besides, there's no point in using TCHAR for URLs, because URLs are ASCII by definition.